nUpdate.WPFUserInterface/UpdaterUI.cs

```diff
@@ -0,0 +1,120 @@
+﻿// Copyright © Sascha Patschka 2018
+
+using System;
+using System.IO;
+using nUpdate.Internal.Core.Localization;
+using nUpdate.ServiceInterfaces;
+using nUpdate.WPFUserInterface.Services;
+
+// ReSharper disable once CheckNamespace
+namespace nUpdate.Updating
+{
+
+    public sealed partial class UpdaterUI
+    {
+        /// <summary>
+        ///     Starts the complete update process and uses the integrated user interface for user interaction.
+        /// </summary>
+        public async void ShowUserInterface()
+        {
+            if (_active)
+                return;
+            _active = true;
+
+            ServiceInjector.InjectServices();
+
+            var dialogService = ServiceContainer.Instance.GetService<IDialogWindowService>();
+            var messageboxService = ServiceContainer.Instance.GetService<IMessageboxService>();
+
+            LocalizationProperties lp = LocalizationHelper.GetLocalizationProperties(UpdateManager.LanguageCulture,
+                UpdateManager.CultureFilePaths);
+
+
+            try
+            {
+                if (!UseHiddenSearch)
+                {
+                    var vmUpdateSearch = new WPFUserInterface.ViewModel.UpdateSearchVm(UpdateManager);
```

Comment:
By adjusting the class namespace, you can use the constructors directly without specifying the namespace explicitly.